hbt/calibration/default.py

```diff
@@ -106,16 +107,18 @@ def default(self: Calibrator, events: ak.Array, **kwargs) -> ak.Array:
             # nominal jec and jer
             events = self[self.jec_nominal_cls](events, **kwargs)
             events = self[self.deterministic_jec_jec_nominal_cls](events, **kwargs)
-            # nominal tec
-            events = self[self.tec_nominal_cls](events, **kwargs)
+            if self.config_inst.campaign.x.year!=2024:
+                # nominal tec
+                events = self[self.tec_nominal_cls](events, **kwargs)
             # nominal ess
             events = self[self.ees_nominal_cls](events, **kwargs)
             # nominal muon scale and resolution
             events = self[self.muon_sr_nominal_cls](events, **kwargs)
-
+    print("year", year := self.config_inst.campaign.x.year)
+    if self.config_inst.campaign.x.year!=2024:
```

Comment:
```suggestion
    # TODO: remove if statement wenn met phi corrections are made available
    if self.config_inst.campaign.x.year != 2024:
```

add TODO to not forget to change it as soon as available